exoseq.nf

```diff
@@ -1,702 +0,0 @@
-#!/usr/bin/env nextflow
```

Comment:
Hmm, I think you've done a `mv` instead of a `git mv` here, meaning that the diff shows a deleted and new file. Nevermind - kind of complicated to fix retrospectively and not worth it.

Reply:
Okay :-)